x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -131,11 +131,6 @@ export function AlertDetails() {
     return <CenterJustifiedSpinner />;
   }
 
-  // Redirect to the 404 page when the user hit the page url directly in the browser while the feature flag is off.
-  if (alertDetail && !isAlertDetailsEnabledPerApp(alertDetail.formatted, config)) {
```

Comment:
we don't have a possibility to delete alert from UI. do you mean deleted by API?

Reply:
Yes, I was thinking about deleting an alert by API as I saw it as a workaround in some SDHs.

Reply:
I added page not found in this [commit](23e523cc7ff129088c4ccba32681eea05d32c35e), but I realized if there is an issue fetching alert details, it will still show `page not found`. I think `Unable to load alert details` is a generic message that will work in both cases - actual error fetching alert details, and alert not found. Wdyt?

I have now updated the code in 364ffe2354ccff58bdf1f20e3d215f0b83a7c1f3 to remove page not found.

Reply:
Great, indeed I think this approach is better, thanks!